package/MDAnalysis/lib/c_distances.pyx

```diff
@@ -20,7 +20,6 @@
 # J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
 #
 #
```

Comment:
can you revert the changes to this file to keep history tidy?

Reply:
Just to make sure, are you asking to commit a revert or to make a separate branch and push force to the current branch, but I guess you have to rebase it afterwards to not include any of the current history.